python/ray/train/predictor.py

```diff
@@ -179,21 +192,21 @@ def _predict_pandas(self, data: "pd.DataFrame", **kwargs) -> "pd.DataFrame":
         raise NotImplementedError
 
     @DeveloperAPI
-    def _predict_arrow(self, data: "pyarrow.Table", **kwargs) -> "pyarrow.Table":
-        """Perform inference on an Arrow Table.
+    def _predict_numpy(
+        self, data: Union[np.ndarray, Dict[str, np.ndarray]], **kwargs
+    ) -> Union[np.ndarray, Dict[str, np.ndarray]]:
+        """Perform inference on a Numpy data.
 
-        Predictors can implement this method instead of ``_predict_pandas``
-        for better performance when the input batch type is a Numpy array, dict of
-        numpy arrays, or an Arrow Table as conversion from these types are zero copy.
+        All DL predictors are expected to implement this method.
```

Comment:
Not necessarily true right? DL predictors just need to override `call_model`

Reply:
The DL predictor here means Torch/Tensorflow that deals with tensor and by default only calls with numpy format.

Custom predictors supposedly only need to override `call_model`